model/implementation/TestRunnerMessageService.php

```diff
@@ -110,4 +120,13 @@ protected function getTerminatedStateMessage(AssessmentTestSession $testSession)
         return parent::getTerminatedStateMessage($testSession);
     }
 
+    private function cantReturnPauseStateMessage(): bool
+    {
+        return $this->getFeatureFlagChecker()->isEnabled(self::FEATURE_FLAG_PROCTOR_NOT_LAUNCH_PAUSE_MESSAGE);
```

Comment:
Please don't use negations in the methods or variable names, it creates a lot of confusion.
Let's reverse this, and name it `isPausedDialogRequired()`. WDYT?
```suggestion
    private function isPausedDialogRequired(): bool
    {
        return !$this->getFeatureFlagChecker()->isEnabled(self::FEATURE_FLAG_PROCTOR_NOT_LAUNCH_PAUSE_MESSAGE);
```

Obviously, the users of the method have to take that into account, and reverse their conditions.